src/components/CodeScreen.tsx

```diff
@@ -25,7 +36,12 @@ export function CodeScreen(props: CodeScreenProps): JSX.Element {
     <>
       <div style={codeScreenStyle}>
         <div style={textStyle}>YOU'VE SELECTED {props.coinSelection}</div>
-        <BackButton label="BACK" handleClick={handleBackBtnClick} />
+        <div style={qrCodeBorderStyle}>
```

Comment:
Separate the QR into its own component.

Reply:
Updated per comment. Created component `PaymentCode`.